src/components/App.js

```diff
@@ -30,7 +29,10 @@ const { appinfo } = Services;
 
 const isMacOS = appinfo.OS === "Darwin";
 
-const verticalLayoutBreakpoint = window.matchMedia("(min-width: 800px)");
+const horizontalLayoutBreakpoint = window.matchMedia("(min-width: 800px)");
```

Comment:
So if we have 3 cases (invisible, vertical, horizontal) with the conditions being (< 10px; 10px - 800px; 800px+) do we  care what layout the debugger is if we cannot see it?

Reply:
That was the intention with this. if it matches positively with either case, then change to that case, if it is below 10px, leave it the way it previously was.